website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,7 +125,22 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
-## Content type
+## Sizes
```

Comment:
This looks like a duplicate from the other size section, maybe an accidental copy/paste?